app/ios/OpenPassport/AppDelegate.mm

```diff
@@ -16,6 +16,16 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   if ([UNUserNotificationCenter class] != nil) {
     UNUserNotificationCenter *center = [UNUserNotificationCenter currentNotificationCenter];
     center.delegate = self;
+
+    // Request permission for notifications
```

Comment:
add support for push notifcations fyi: @remicolin @motemotech

Reply:
going to enable this after we configure the APN ; this code causes the user to prompted to enable app notifications